deployment_scripts/README.md

```diff
@@ -51,5 +51,4 @@ After downloading that script, execute it in a shell. The first argument should
 - `./deploy_linux.sh "/home/test/monkey"` (deploys under /home/test/monkey)
 - `./deploy_linux.sh "" "master"` (deploys master branch in script directory)
 - `./deploy_linux.sh "/home/user/new" "master"` (if directory "new" is not found creates it and clones master branch into it)
-
-You may also pass in an optional third `false` parameter to disable downloading the latest agent binaries.
\ No newline at end of file
+- `./deploy_linux.sh "" "" "v1.7.1" ` (If you want to download any specific version of the releases, Provide the version in the 3rd argument, Else by Default, it will download the latest release of binaries)
```

Comment:
lowercase p, e and D, or change the grammer.

Reply:
Sure 👍.